action.yml

```
@@ -7,12 +7,22 @@ inputs:
 
       By default, the `github.token` is configured. See https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
       for a description of the default token's permissions.
+  extra_nix_config:
+    description: |
+      Additional configuration text to append to nix.conf. See https://docs.lix.systems/manual/lix/2.90/command-ref/conf-file.html
+      for a description of the config file format.
 runs:
   using: "composite"
   steps:
+    - name: Cache installer
+      uses: actions/cache
```

```
Expected format {org}/{repo}[/path]@ref. Actual 'actions/cache'
```

And yeah, we should be pinning at the very least to a major version.

```suggestion
      uses: actions/cache@v4
```